### ISSUE TITLE
Dependency "com.github.Dansoftowner:jSystemThemeDetector:3.9.1." is missing

### ISSUE THREAD

**Issue by ClaudiusSeibt** [tudalgo/fopbot-playground#15]
Error thrown by InteliJ:

```
Execution failed for task ':fopbot_playground.example.FillRectangle.main()'.
Could not resolve all files for configuration ':runtimeClasspath'.
   Could not find com.github.Dansoftowner:jSystemThemeDetector:3.9.1.
     Searched in the following locations:
       - https://repo.maven.apache.org/maven2/com/github/Dansoftowner/jSystemThemeDetector/3.9.1/jSystemThemeDetector-3.9.1.pom
     Required by:
         root project : > org.tudalgo:fopbot:0.8.0

Possible solution:
 - Declare repository providing the artifact, see the documentation at https://docs.gradle.org/current/userguide/declaring_repositories.html
```

**Comment by ClaudiusSeibt** [comment]
The url "[https://repo.maven.apache.org/maven2/com/github/Dansoftowner/jSystemThemeDetector/3.9.1/jSystemThemeDetector-3.9.1.pom](https://repo.maven.apache.org/maven2/com/github/Dansoftowner/jSystemThemeDetector/3.9.1/jSystemThemeDetector-3.9.1.pom)" returns a 404 status code

**Comment by ClaudiusSeibt** [comment]
The code runs fine before this commit: [https://github.com/tudalgo/fopbot-playground/commit/00495f06a87d40fd367eb8049b0e616fcfd63a36](https://github.com/tudalgo/fopbot-playground/commit/00495f06a87d40fd367eb8049b0e616fcfd63a36), so just revert back to this commit for now: [https://github.com/tudalgo/fopbot-playground/commit/eaf8b19f4361020f6066f45e661e12b4de2a8c2f](https://github.com/tudalgo/fopbot-playground/commit/eaf8b19f4361020f6066f45e661e12b4de2a8c2f), that worked for me

**Comment by Rdeisenroth** [comment]
Fixed in [2ec3fc6](https://github.com/tudalgo/fopbot-playground/commit/2ec3fc6713f289b2278fc1de0997fc99264bc7e7)
